templates/flavors/README.md

```diff
@@ -14,3 +14,77 @@ appended to the end of the cluster-template.yaml, e.g cluster-template-{director
 flavor can be used by specifying `--flavor {directory-name}`.
 
 To generate all CAPZ flavors, run `make generate-flavors`.
+
+
+## Running flavor clusters as a tilt resource
```

Comment:
let's put a link to these instructions in https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/docs/development.md#using-tilt

Reply:
Thanks for the help!  I just added new commit for this one.  Let me know your thoughts!